src/System.Runtime.Extensions/tests/System/StringComparer.cs

```diff
@@ -51,6 +53,13 @@ private static void VerifyComparer(StringComparer sc, bool ignoreCase)
 
             Assert.Equal(ignoreCase, sc.Equals(s1, s1b));
             Assert.Equal(ignoreCase, ((IEqualityComparer)sc).Equals(s1, s1b));
+            
+            Assert.NotEqual(sc.GetHashCode(aa), sc.GetHashCode(bb));
```

Comment:
This one needs a RemoteInvoke (with several attempts if necessary) since there is a 1/4billion chance it will match by accident. A child process will change the random seed of the randomized hash code, so you can repeatedly use a child process until it is not equal. And only 1/4 billion chance you will need to do it more than once.

Reply:
As we discussed offline I believe there is no need for RemoteInvoke. We are not testing random hashing here. We are just comparing GetHashCode values for two different strings.

Reply:
What guarantees that two different strings won't have the same hash code?